enhancements/update/cvo-log-level-api.md

```diff
@@ -0,0 +1,213 @@
+And potentially, in the future, the HyperShift may even dynamically utilize this property.
+
+### Risks and Mitigations
+
+No risks are known.
+
+### Drawbacks
+
+No drawbacks are known.
+
+## Design Details
+
+### Open Questions [optional]
+
+No open questions.
+
+### Test Plan
+
+Unit tests will be written to test if setting `operatorLogLevel` sets the respective logging in the CVO.
+
+### Graduation Criteria
+
+This will be released directly to GA. The new field and its data type are equal to that of the [`OperatorLogLevel`](https://github.com/openshift/api/blob/1f9525271dda5b7a3db735ca1713ad7dc1a4a0ac/operator/v1/types.go#L74) field in the [`OperatorSpec`](https://github.com/openshift/api/blob/1f9525271dda5b7a3db735ca1713ad7dc1a4a0ac/operator/v1/types.go#L54) 
+structure of the [`operator/v1`](https://github.com/openshift/api/tree/1f9525271dda5b7a3db735ca1713ad7dc1a4a0ac/operator/v1) 
+package. This field in the [`operator/v1`](https://github.com/openshift/api/tree/1f9525271dda5b7a3db735ca1713ad7dc1a4a0ac/operator/v1) package has been stable for several years. 
+Thus, the addition may be directly released to GA.
+
+#### Dev Preview -> Tech Preview
+
+This will be released directly to GA.
+
+#### Tech Preview -> GA
+
+This will be released directly to GA.
+
+#### Removing a deprecated feature
+
+No existing feature will be deprecated or removed.
+
+### Upgrade / Downgrade Strategy
+
+N/A
+
+### Version Skew Strategy
+
+The relevant components for this enhancement are the CVO and the ClusterVersion CRD. These components move closely together on updates and downgrades. During a version skew, the default log level will be used.
+
+A newer CVO consuming an older ClusterVersion will receive an empty `OperatorLogLevel` field, and the CVO will continue using the default log level. An older CVO consuming newer ClusterVersion will not notice the `OperatorLogLevel` field, and the CVO will continue using the default log level.
+
+### Operational Aspects of API Extensions
+
+This enhancement proposes a minor addition to the already existing ClusterVersion CRD.
+This new addition will operationally impact only the CVO.
+
+#### Failure Modes
+
+N/A
+
+#### Support Procedures
+
+N/A
+
+## Implementation History
+
+Empty as of this moment.
+
+## Alternatives
+
+* The proposed solution doesn't strictly follow the DRY principle as the `LogLevel` type is copied over to the `github.com/openshift/api/config/v1` package to solve the import cycle. Another solution could be moving the data type `LogLevel` from its package `github.com/openshift/api/operator/v1` to a newly created package or to the `github.com/openshift/api/config/v1` package.
```

Comment:
I am a little bit impartial regarding what option is better. Any feedback or opinions are welcomed :+1: